frontend/app/lib/open-payments.server.ts

```diff
@@ -0,0 +1,318 @@
+import {
+  type AuthenticatedClient,
+  type PendingGrant,
+  type Quote,
+  type WalletAddress,
+  createAuthenticatedClient,
+  isFinalizedGrant,
+  isPendingGrant
+} from '@interledger/open-payments'
+import { createId } from '@paralleldrive/cuid2'
+import { randomUUID } from 'crypto'
```

Comment:
Nit: Maybe use the global `crypto.randomUUID()`? That way we don't have to over-rely on Node env in all the files. Helps with migration.